src/components/TaskGroupProgress/index.jsx

```diff
@@ -244,13 +245,29 @@ export default class TaskGroupProgress extends Component {
     }
   };
 
+  getFaviconStatus = statusCount => {
+    const { failed, pending, running, unscheduled } = statusCount;
+    const incompletedTasks = pending + running + unscheduled;
+
+    if (failed > 0) {
+      return 'Failed';
```

Comment:
Instead of hard-coding these values, you can use `TASK_STATE.FAILED`, `TASK_STATE.RUNNING`, and `TASK_STATE.COMPLETED`. These can be imported from `src/utils/constants`.